handlers/http.go

```diff
@@ -35,8 +39,9 @@ func stringifyRequest(req common.Request) *bytes.Buffer {
 }
 
 type httpRateLimiter struct {
-	rateLimiter ratelimiter.RateLimiter
-	proxy       http.Handler
+	rateLimiter  ratelimiter.RateLimiter
+	proxy        http.Handler
+	AllowOnError bool // Do not limit on errors when true
```

Comment:
Maybe a setter? I didnt want to change the constructor signature and I need to be able to inject the flag.